<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnvSetupPage</class>
 <widget class="QWidget" name="EnvSetupPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>New Application Setup</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18pt; font-weight: bold;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="instructionLabel">
     <property name="text">
      <string>Define the root folder and initialize a Git repository for the new target application.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="pathLabel">
       <property name="text">
        <string>Target Project Root Folder:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="projectPathLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="confirmPathButton">
     <property name="text">
      <string>Confirm Project Folder</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="gitLabel">
     <property name="text">
      <string>Git repository not initialized.</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: orange;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="initGitButton">
     <property name="text">
      <string>Initialize Git Repository</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="proceedButton">
     <property name="text">
      <string>Confirm Setup &amp; Proceed to Specification</string>
     </property>
    </widget>
   </item>
  <item>
    <spacer name="bottomVerticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>